### ISSUE TITLE
A little question about Optimizer params

### ISSUE THREAD

**Issue by Mendel1** [novioleo/crnn.mxnet#9]
@novioleo Hi, I'm trying to train crnn on synth90k with your code.But it behaves badly.While the origin version get  93% on that dataset ,my model can only get 60%  accuracy. A little different from the original version , I use the adam optimizer as  you used  with learning rate set to be 1e-4.Could you give me some advice?
Thank you  very much.

**Comment by Mendel1** [comment]
@novioleo Thank you for your answer.I tried the origin version with crnn.py. And I found the  adadelta optimizer in origin version doesn't work for me while adam can achieve 0.60 accuracy. So could you give me some advice on how to adjust the params to improve its behavior? I also tried the origin torch version .And it's amazing that it achieve 0.93 accuracy with params of adadelta optimizer set defaultly. Is there any difference between the optimizer of mxnet and torch?

**Comment by Mendel1** [comment]
By the way, why you use lstm defined by yourself instead of the lstmcell in mxnet? Is there any differece?

**Comment by novioleo** [comment]
there is no difference of adam between torch or mxnet,i suggest you try to figure out the difference of the convolution layers,maybe,i changed but i forgot~ sorry.

by the other hand,the lstmcell i defined in my code is follow the official example,the lstm of gluon or mxnet is not mature when i wrote this code.I suppose there are some differences between these.

**Comment by novioleo** [comment]
卷积层我当时改过，你可以参考下pytorch的将这个改回来。
例如卷积的relu我换成了leaky-relu，是为了适应我的数据。你可能需要根据你的数据适当改下你的模型。

**Comment by Mendel1** [comment]
刚刚在图书馆看到忍不住笑出声了。上面还是英文交流，说着说着就是国语了。:joy:谢谢了，我再改改看看。

**Comment by novioleo** [comment]
唔，你可以来群里。我们qq群里说。readme里有群号。
